src/showcase/index.md

```diff
@@ -42,29 +46,51 @@ toc: false
             {% if modulo3 == '0' %}
-                <div class="showcase__apps__cards text-left card-deck">
+                <div class="showcase__apps__cards text-left card-deck flex-column flex-lg-row">
             {% endif %}
                     <div class="card">
+                        <div class="card-header">
+                            {% if case.logo_src %}
+                                {% asset '{{ case.logo_src }}' class='showcase__apps__logo' %}
+                            {% endif %}
+                        </div>
                         <div class="card-body">
                             <h3>{{ case.name }}</h3>
                             <p>{{ case.description }}</p>
                         </div>
                         <div class="card-footer">
-                            <a class="btn btn-link" href="{{ case.learn_more_link }}">Learn more</a>
+                            {% if case.learn_more_link %}<a href="{{ case.learn_more_link }}">Learn more</a>{% endif %}
+                            {% if case.play_store_link or case.app_store_link %}
+                                <span class="dropdown">
+                                    <a href="#" class="dropdown-toggle" id="dropdownMenuButton{{ forloop.index }}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                        Download
+                                    </a>
+                                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton{{ forloop.index }}">
+                                        {% if case.play_store_link %}<a class="dropdown-item" href="{{ case.play_store_link }}" target="_blank">Android</a>{% endif %}
+                                        {% if case.app_store_link %}<a class="dropdown-item" href="{{ case.app_store_link }}" target="_blank">iOS</a>{% endif %}
+                                    </div>
+                                </span>
+                            {% endif %}
                         </div>
                     </div>
             {% if modulo3 == '2' or forloop.last %}
                 </div>
             {% endif %}
         {% endfor %}
+        
+        <div class="row">
+            <div class="col-12 text-center">
+                <a class="landing-page__cta__button btn btn-primary" href="https://itsallwidgets.com/" target="_blank">Explore more apps built with Flutter</a>
+            </div>
+        </div>
     </div>
 </section>
 
 <section class="landing-page__cta card text-center">
     <div class="card-body">
         <h2 class="landing-page__cta__headline">Try Flutter today.</h2>
         <p class="landing-page__cta__body">Getting started is easy.</p>
-        <a class="landing-page__cta__button btn btn-primary" href="/get-started/install">Get started</a>
+        <a class="landing-page__cta__button btn btn-primary btn-cta" href="/get-started/install">Get started</a>
```

Comment:
Compare these two buttons from the page:

- <img width="366" alt="screen shot 2018-10-11 at 02 36 58" src="https://user-images.githubusercontent.com/4140793/46785069-9ea25b80-ccfe-11e8-9d3f-80c701a52400.png">
- <img width="290" alt="screen shot 2018-10-11 at 02 37 03" src="https://user-images.githubusercontent.com/4140793/46785076-a3ffa600-ccfe-11e8-8809-322b4f52ef67.png">

The text sizes are quite similar but the "Get started" button itself is much larger. IMHO, the "Get started" text should be proportionally larger too.